api/auth/authorize.ts

```diff
@@ -48,7 +48,8 @@ export default async function authorize(
       identity.email
         ?.substring(0, identity.email?.indexOf("@"))
         .replace(/[^a-zA-Z._]/g, "_")
-        .replace(/__/g, "_") ||
+        .replace(/__/g, "_")
+        .replace(/\./g, "_") ||
```

Comment:
@njhargis if you'd like to exclude dot characters from usernames, removing it from line 50 above might be a better tweak `/[^a-zA-Z._]/g` => `/[^a-z0-9_]/gi`